src/platform/packages/shared/kbn-unified-tabs/src/components/tabbed_content/tabbed_content.tsx

```diff
@@ -39,6 +39,7 @@ export interface TabbedContentProps extends Pick<TabsBarProps, 'maxItemsCount'>
   createItem: () => TabItem;
   onChanged: (state: TabbedContentState) => void;
   getPreviewData: (item: TabItem) => TabPreviewData;
+  onEvent: (eventName: string, payload?: TabsEBTPayload) => void;
```

Comment:
Nit: I'd probably go with something like `onEBTEvent` or similar for clarity.

Reply:
Addressed in [3d7ab45](https://github.com/elastic/kibana/pull/234344/commits/3d7ab45042b8938b58454af141870b62794a6236)